pkg/cmd/login.go

```diff
@@ -68,8 +68,17 @@ var loginCmd = &cobra.Command{
 			fmt.Println("Waiting...")
 		}
 
+		// auth flow must complete within 5 minutes
+		timeout := 5 * time.Minute
+		completeBy := time.Now().Add(timeout)
+
 		response = nil
 		for {
+			// we don't respect --no-timeout here
+			if time.Now().After(completeBy) {
+				utils.HandleError(fmt.Errorf("login timed out after %d minutes", int(timeout.Minutes())))
```

Comment:
Can we capitalize "login"?

Reply:
This is lowercase for consistency with our other errors. `HandleError()` will prepend `Error:` to this, so the printed text will read `Error: login timed out after 5 minutes`.

Reply:
gotcha